RSI

```diff
@@ -15,8 +13,21 @@ start_time = datetime.datetime(2020, 10, 1)
 #end_time = datetime.datetime(2020, 1, 20)
 end_time = datetime.datetime.now().date().isoformat()         # today
 
-df = ticker_df
+connected = False
+while not connected:
+    try:
+        ticker_df = web.get_data_yahoo(ticker, start=start_time, end=end_time)
+        connected = True
+        print('connected to yahoo')
+    except Exception as e:
+        print("type error: " + str(e))
+        time.sleep( 5 )
+        pass
 
+ticker_df = ticker_df.reset_index()
+print(ticker_df.head(5))
+
+df = ticker_df
 
 def computeRSI(data, time_window):
```

Comment:
why 0 * diff ?

Reply:
Im trying to figure it out as well, it didn't work as intended without the 0 * diff

Reply:
try logic similar to this! You're on the right path! 

up, down = data.copy(), data.copy() #copy data twice to split 
up[up < 0] = 0 #concerning only positive returns 
down[down > 0] = 0 #concerning only negative returns 

#Calculating SMA 
roll_up2 = up.rolling(21).mean() #SMA for pos returns data  
roll_down2 = down.abs().rolling(21).mean() #SMA for neg returns data

Reply:
Ohh, i figured it out.
When multiplying with 0, all values become zero, but the sign of the original data (either positive or negative) still remains as a very small decimal number.
So the following code 
    up_chg[diff > 0] = diff[diff > 0]
    down_chg[diff < 0] = diff[diff < 0]
helps assign the positive and negative values respectively back to the dataframes up_chg and down_chg

Reply:
Do fix accordingly and update this script! Great job so far. I will merge once this change is implemented. If possible, please do a quick visualisation of your results with matplotlib!